src/howler.core.js

```diff
@@ -1479,7 +1479,6 @@
         // Remove the source or disconnect.
         if (!self._webAudio) {
           // Set the source to 0-second silence to stop any downloading.
-          sounds[i]._node.src = 'data:audio/wav;base64,UklGRigAAABXQVZFZm10IBIAAAABAAEARKwAAIhYAQACABAAAABkYXRhAgAAAAEA';
```

Comment:
A better solution would be to only not run this on IE. 

@fabioelia do you think that would work better?